src/init/kafka_exporter.rs

```diff
@@ -0,0 +1,347 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::exporters::kafka::config::{
+    AcknowledgementMode, KafkaExporterConfig, PartitionerType, SerializationFormat,
+};
+use crate::init::parse::parse_key_val;
+use clap::{Args, ValueEnum};
+use serde::Deserialize;
+
+#[derive(Debug, Args, Clone, Deserialize)]
+pub struct KafkaExporterArgs {
+    /// Kafka broker addresses (comma-separated)
+    #[arg(
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_BROKERS",
+        default_value = "localhost:9092"
+    )]
+    pub kafka_exporter_brokers: String,
```

Comment:
In addition, for multiple exporter configuration, the struct members need to be renamed without the `kafaka_exporter_` prefix and then using the `long()` argument for clap to specify the full version:
```
#[arg(
        long("kafka-exporter-brokers"),
        env = "ROTEL_KAFKA_EXPORTER_BROKERS",
        default_value = "localhost:9092"
    )]
pub brokers: String,

```

Reply:
If you find that clap throws an error talking about argument collision, then you may need to add a custom `id` to the arg like `id("KAFKA_EXPORTER_COMPRESSION")`. I had to do that here for Clickhouse endpoint:
https://github.com/streamfold/rotel/blob/c50761b13a64b6728f91ec372dd6f03849bca45d/src/init/clickhouse_exporter.rs#L9C2-L14C33